File: guanabara-java/src/com/danillo/algoritmos/lista10/Exercicio98.java
package com.danillo.algoritmos.lista10;

import java.util.Scanner;

public class Exercicio98 {
    public static int superSomador(int a, int b){
        int soma = 0;
        for (int i = a; i < b; i++){
            soma = soma + i;
        }
        return soma;
    }

    public static void main(String[] args){
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira um número: ");
        int a = entrada.nextInt();

        System.out.println("Insira outro número: ");
        int b = entrada.nextInt();

        int resultado = superSomador(a, b);
        System.out.println("Resultado: " +resultado);
    }
}

File: guanabara-python/src/lista10/exercicio98.py
def super_somador(a, b):
    soma = 0
    for i in range(a, b):
        soma = soma + i
    return soma


a = int(input("Insira um número: "))
b = int(input("Insira outro número: "))
resultado = super_somador(a, b)
print(resultado)
